src/runtime/audience-action-flow-test.js

```diff
@@ -1339,69 +1339,159 @@ describes.realWin('AudienceActionIframeFlow', (env) => {
     dialogManagerMock.verify();
   });
 
-  it('opens dialog with closeOnBackgroundClick=false by default', async () => {
-    const audienceActionFlow = new AudienceActionIframeFlow(runtime, {
-      action: 'TYPE_REGISTRATION_WALL',
-      configurationId: 'configId',
-      onCancel: onCancelSpy,
-      autoPromptType: AutoPromptType.SUBSCRIPTION,
+  describe('closeOnBackgroundClick experiment active', () => {
+    after(() => {
+      dialogManagerMock.verify();
```

Comment:
Please move to individual tests; expectations in the `after` method are an anti-pattern, new instances should not be added.